src/background/services/monetization.ts

```diff
@@ -434,11 +434,22 @@ export class MonetizationService {
 
     const { oneTimeGrant, recurringGrant, ...dataFromStorage } = storedData;
 
+    // #1071
+    let minSendAmount = '0';
+      if (tab.id) {
+        // If we have a payment manager for this tab, get the min send amount
+        const paymentManager = this.tabState.paymentManagers.get(tab.id);
+        if (paymentManager) {
+          minSendAmount = paymentManager.minSendAmount.toString();
+        }
+      }
+
     return {
       ...dataFromStorage,
       balance: balance.total.toString(),
       tab: this.tabState.getPopupTabData(tab),
       transientState: this.storage.getPopupTransientState(),
+      minSendAmount,
```

Comment:
Given this data is specific current tab, can you move this to `tab` key's value? Tab info changes often.